Admin_Registration/README.md

```diff
@@ -0,0 +1,64 @@
+# Admin Registration
+
+## Type
+
+Improper Input Validation
+
+## Description
+
+Register as a user with administrator privileges.

```

Comment:
in the best case you also add some more information like "i want to try creating a user with admin priviliges by abusing improper input validation in the system and manipulate non-userland information in the request before the backend receives it. [...]" 

--> description in this file should be a full description of what you do and how you do that, but without too much details

Reply:
Thats the description of challenge in Juice Shop